### PR TITLE
[Security Content] Add Investigation Guides to Windows rules

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Adds IGs to the following rules:

- Potential Shadow Credentials added to AD Object
- Suspicious Process Execution via Renamed PsExec Executable
- Unusual Network Activity from a Windows System Binary
- System Information Discovery via Windows Command Shell
- PowerShell Mailbox Collection Script
- PowerShell Suspicious Script with Clipboard Retrieval Capabilities
- LSASS Memory Dump Creation
- Wireless Credential Dumping using Netsh Command
- Suspicious Antimalware Scan Interface DLL
- Potential Antimalware Scan Interface Bypass via PowerShell

Minor adjusts in the following rules' IGs:
- Exporting Exchange Mailbox via PowerShell
- Modification of AmsiEnable Registry Key
- PowerShell Script Block Logging Disabled

### CODE REVIEW

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 1]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Unusual Network Activity from a Windows System Binary"
+note = """## Triage and analysis
+
+### Investigating Unusual Network Activity from a Windows System Binary
+
+Attackers can abuse certain trusted developer utilities to proxy the execution of malicious payloads. As those utilities are usually signed, they can bypass security controls in place that could prevent or detect direct execution.
+
+This detection rule identifies network connections done by these utilities, which can indicate abuse to execute payloads or the masquerading as one of these processes.
```
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:26]:
```suggestion
This detection rule checks for trusted developer utilities connecting to networks as this behavior can indicate abuse to executed payloads or process masquerading.
```

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 2]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will see unrendered Markdown in this guide.
```
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:29]:
```suggestion
> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. If you are using an older version of the Stack, Markdown text in this guide will not render.
```

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 3]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Investigate abnormal behaviors observed by the subject process, such as registry or file modifications, and any spawned child processes.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Examine the host for derived artifacts that indicates suspicious activities:
+  - Analyze the process executable using a private sandboxed analysis system.
+  - Observe and collect information about the following activities in both the sandbox and the alert subject host:
+    - Attempts to contact external domains and addresses.
+      - Use the Elastic Defend network events to determine domains and addresses contacted by the subject process by filtering by the process' `process.entity_id`.
+      - Examine the DNS cache for suspicious or anomalous entries.
```
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:41]: Same suggestion that made [here](https://github.com/elastic/detection-rules/pull/2521/files#diff-783799e1c367669274ab7c6261c7331663ee4d7cfcb12f3f297879303e1923baR43).

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 4]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+        - !{osquery{"query":"SELECT * FROM dns_cache", "label":"Osquery - Retrieve DNS Cache"}}
+    - Use the Elastic Defend registry events to examine registry keys accessed, modified, or created by the related processes in the process tree.
+    - Examine the host services for suspicious or anomalous entries.
```
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:44]: Same suggestion made [here](https://github.com/elastic/detection-rules/pull/2521/files#diff-783799e1c367669274ab7c6261c7331663ee4d7cfcb12f3f297879303e1923baR46).

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 5]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+      - !{osquery{"query":"SELECT concat('https://www.virustotal.com/gui/file/', sha1) AS VtLink, name, description, start_type, status, pid, services.path FROM services JOIN authenticode ON services.path = authenticode.path OR services.module_path = authenticode.path JOIN hash ON services.path = hash.path WHERE authenticode.result != 'trusted'","label":"Osquery - Retrieve Service Unsigned Executables with Virustotal Link"}}
+  - Retrieve the files' SHA-256 hash values using the PowerShell `Get-FileHash` cmdlet and search for the existence and reputation of the hashes in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+
+### False positive analysis
+
+- As trusted developer utilities have dual-use purposes, alerts derived from this rule are not essentially malicious. If these utilities are contacting internal or known trusted domains, review their security and consider adding an exception if the domain is safe.
```
Comment (nastasha-solomon) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:52]:
```suggestion
- As trusted developer utilities have dual-use purposes, alerts derived from this rule are not essentially malicious. If these utilities are contacting internal or known trusted domains, review their security and consider creating exceptions if the domain is safe.
```

File: rules/windows/execution_suspicious_psexesvc.toml [hunk 6]
```diff
@@ -17,14 +17,45 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Process Execution via Renamed PsExec Executable"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating PsExec Network Connection
+
+PsExec is a remote administration tool that enables the execution of commands with both regular and SYSTEM privileges on Windows systems. It operates by executing a service component, "Psexecsvc", on the remote system, which then runs the specified process and returns the results to the local system. Microsoft develops PsExec as part of the Sysinternals Suite. Although commonly used by administrators, PsExec is frequently used by attackers to enable lateral movement and execute commands as SYSTEM to disable defenses and bypass security protections.
```
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_psexesvc.toml:24]:
```suggestion
PsExec is a remote administration tool that enables the execution of commands with both regular and SYSTEM privileges on Windows systems. It operates by executing the service component `Psexecsvc` on a remote system, which then runs a specified process and returns the results to the local system. Microsoft developed PsExec as part of the Sysinternals Suite. Although commonly used by administrators, PsExec is frequently used by attackers to enable lateral movement and execute commands as SYSTEM to disable defenses and bypass security protections.
```

File: rules/windows/execution_suspicious_psexesvc.toml [hunk 7]
```diff
@@ -17,14 +17,45 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
+- Investigate what commands were run, and assess whether this behavior is prevalent in the environment by looking for similar occurrences across hosts.
+
+### False positive analysis
+
+- This mechanism can be used legitimately. As long as the analyst did not identify suspicious activity related to the user or involved hosts, and the tool is allowed by the organization's policy, such alerts can be dismissed.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+  - Prioritize accordingly with the role of the servers and users involved.
```
Comment (nastasha-solomon) [inline rules/windows/execution_suspicious_psexesvc.toml:43]: I'm not quite sure what's being suggested here. Are we recommending to prioritize cases involving servers and users? Or are we suggesting to prioritize based on the importance of certain server or user roles? Or are we suggesting something else entirely?

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for all the work you've done with these guides, @w0rk3r ! I left some suggestions and questions for you below. Hopefully they're helpful :)

File: rules/windows/collection_email_powershell_exchange_mailbox.toml [hunk 8]
```diff
@@ -48,6 +50,7 @@ Attackers can abuse this functionality in preparation for exfiltrating contents,
 ### Response and remediation
 
 - Initiate the incident response process based on the outcome of the triage.
+  - If PII or other classified data is involved, investigations into this should be prioritized.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/collection_email_powershell_exchange_mailbox.toml:53]: Duplicated one, nice catch

File: rules/windows/collection_posh_mailbox.toml [hunk 9]
```diff
@@ -17,14 +17,71 @@ index = ["winlogbeat-*", "logs-windows.*"]
...
+- Prioritize cases that involve personally identifiable information (PII) or other classified data.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are identified. Reset passwords for these accounts and other potentially compromised credentials, such as email, business systems, and web services.
+- Run a full antimalware scan. This may reveal additional artifacts left in the system, persistence mechanisms, and malware components.
+- Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
+- Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
+
+## Setup
+
+The 'PowerShell Script Block Logging' logging policy must be enabled.
+Steps to implement the logging policy with with Advanced Audit Configuration:
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/collection_posh_mailbox.toml:62]: Not in this case, kibana don't need two line breaks anymore

File: rules/windows/credential_access_lsass_memdump_file_created.toml [hunk 10]
```diff
@@ -18,15 +18,66 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
+note = """## Triage and analysis
+
+### Investigating LSASS Memory Dump Creation
+
+Local Security Authority Server Service (LSASS) is a process in Microsoft Windows operating systems that is responsible for enforcing security policy on the system. It verifies users logging on to a Windows computer or server, handles password changes, and creates access tokens.
+
+This rule looks for the creation of memory dump files with file names compatible with credential dumping tools or which start with `lsass`.
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic stack version 8.5.0. Older Elastic stacks versions will see unrendered markdown in this guide.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_lsass_memdump_file_created.toml:30]: I'll adjust it to the verbiage from https://github.com/elastic/detection-rules/pull/2513

File: rules/windows/credential_access_lsass_memdump_file_created.toml [hunk 11]
```diff
@@ -18,15 +18,66 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
+- Identify the process responsible for creating the dump file.
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Identify the user account that performed the action and whether it should perform this kind of action.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Examine the host for derived artifacts that indicates suspicious activities:
+  - Analyze the process executable using a private sandboxed analysis system.
+  - Observe and collect information about the following activities in both the sandbox and the alert subject host:
+    - Attempts to contact external domains and addresses.
+      - Use the Elastic Defend network events to determine domains and addresses contacted by the subject process by filtering by the process' `process.entity_id`.
+      - Examine the DNS cache for suspicious or anomalous entries.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_lsass_memdump_file_created.toml:43]: I'll evaluate a change in this structure soon, as we will need to accommodate more plugins here, but I'll maintain like this rn

File: rules/windows/credential_access_wireless_creds_dumping.toml [hunk 12]
```diff
@@ -14,9 +14,46 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
 name = "Wireless Credential Dumping using Netsh Command"
-note = """## Setup
+note = """## Triage and analysis
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+### Investigating Wireless Credential Dumping using Netsh Command
+
+Netsh is a Windows command-line tool used for network configuration and troubleshooting, which enables the management of network settings and adapters, wireless network profiles, and other network-related tasks.
+
+This rule looks for the patterns used to dump credentials from wireless network profiles using Netsh, which can enable attackers to bring their own devices to the network.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_wireless_creds_dumping.toml:23]: Once they dump credentials using Netsh, it means they can just login into wireless networks with their devices

File: rules/windows/execution_suspicious_psexesvc.toml (hunk 7 above)
Comment (w0rk3r) [reply rules/windows/execution_suspicious_psexesvc.toml:43]: Yeah, to prioritize based on the how critical/important they are

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 13]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Unusual Network Activity from a Windows System Binary"
+note = """## Triage and analysis
+
+### Investigating Unusual Network Activity from a Windows System Binary
+
+Attackers can abuse certain trusted developer utilities to proxy the execution of malicious payloads. Since these utilities are usually signed, they can bypass the security controls that were put in place to prevent or detect direct execution.
+
+This rule identifies network connections done by trusted developer utilities, which can indicate abuse to execute payloads or process masquerading.
```
Comment (joepeeples) [inline rules/windows/defense_evasion_network_connection_from_windows_binary.toml:26]: Nit word choice: maybe "established" or "created"?
```suggestion
This rule identifies network connections established by trusted developer utilities, which can indicate abuse to execute payloads or process masquerading.
```

File: rules/windows/execution_suspicious_psexesvc.toml (hunk 7 above)
Comment (joepeeples) [reply rules/windows/execution_suspicious_psexesvc.toml:43]: Maybe something like this?
```suggestion
  - Prioritize cases involving critical servers and users.
```

**Review by joepeeples — COMMENTED** [review]
Great content, thanks for preparing! Just a few small edits here and there.

**Review by imays11 — COMMENTED** [review]
Did you intentionally only update ```update_date``` for the rules where you added a new investigation guide, and not for the rules that already had an investigation guide that you just edited?

File: rules/windows/credential_access_wireless_creds_dumping.toml [hunk 14]
```diff
@@ -14,9 +14,46 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+### Investigating Wireless Credential Dumping using Netsh Command
+
+Netsh is a Windows command line tool used for network configuration and troubleshooting. It enables the management of network settings and adapters, wireless network profiles, and other network-related tasks.
+
+This rule looks for patterns used to dump credentials from wireless network profiles using Netsh, which can enable attackers to bring their own devices to the network.
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic stack version 8.5.0. Older Elastic stacks versions will see unrendered markdown in this guide.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_wireless_creds_dumping.toml:26]: https://github.com/elastic/detection-rules/pull/2513/commits/464bf34f01fbe3145b41eb0dd7ed6cc20e2633b4

File: rules/windows/defense_evasion_network_connection_from_windows_binary.toml [hunk 15]
```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
...
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will see unrendered Markdown in this guide.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/defense_evasion_network_connection_from_windows_binary.toml:29]: https://github.com/elastic/detection-rules/pull/2513/commits/464bf34f01fbe3145b41eb0dd7ed6cc20e2633b4

File: rules/windows/credential_access_lsass_memdump_file_created.toml [hunk 16]
```diff
@@ -18,15 +18,66 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
...
+note = """## Triage and analysis
+
+### Investigating LSASS Memory Dump Creation
+
+Local Security Authority Server Service (LSASS) is a process in Microsoft Windows operating systems that is responsible for enforcing security policy on the system. It verifies users logging on to a Windows computer or server, handles password changes, and creates access tokens.
+
+This rule looks for the creation of memory dump files with file names compatible with credential dumping tools or that start with `lsass`.
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will see unrendered Markdown in this guide.
```
[the remark this answers is not in the document]
Comment (w0rk3r) [reply rules/windows/credential_access_lsass_memdump_file_created.toml:30]: https://github.com/elastic/detection-rules/pull/2513/commits/464bf34f01fbe3145b41eb0dd7ed6cc20e2633b4

**Comment by w0rk3r** [comment]
@imays11 yes

**Comment by w0rk3r** [comment]
@joepeeples @nastasha-solomon can I have a final review on this one?

**Review by imays11 — APPROVED** [review]

**Comment by imays11** [comment]
> @imays11 yes, as these are just small nits

okay, then it looks good to me 👍🏾

**Review by terrancedejesus — APPROVED** [review]
LGTM. Great work!

**Review by joepeeples — APPROVED** [review]
LGTM, thanks for this great work!

**Pull request merged**